support-frontend/assets/helpers/abTests/__tests__/landingPageAbTests.spec.ts

```diff
@@ -0,0 +1,125 @@
+import type { LandingPageTest } from '../../globalsAndSwitches/landingPageSettings';
+import {
+	fallBackLandingPageSelection,
+	getLandingPageParticipations,
+	getLandingPageVariant,
+} from '../landingPageAbTests';
+import { LANDING_PAGE_PARTICIPATIONS_KEY } from '../sessionStorage';
+
+const ukTest: LandingPageTest = {
```

Comment:
Could this be renamed to non-US Test as we're passing in all the country groups other than the US here? Or we could just pass in the GBPCountries